core/src/main/resources/AppConstants.properties

```diff
@@ -50,6 +50,7 @@ upload.dir=${ibis.tmpdir}
 
 flow.adapter.dir=${ibis.tmpdir}/flow/adapter
 flow.config.dir=${ibis.tmpdir}/flow/config
+javascript.engines=nl.nn.adapterframework.extensions.javascript.J2V8,nl.nn.adapterframework.extensions.javascript.Nashorn
```

Comment:
Since this property is solely for the flow diagrams maybe change it to `flow.javascript.engine`